lineup.js

```diff
@@ -1,9 +1,14 @@
 function calculateTotalSalary(lineup) {
-  return 0
+  return lineup.reduce((playerSalary, player) => {
```

Comment:
Small naming thing. The accumulator isn't a single player's salary which is what your name implies. It's building up to the total of all salaries so something like `total` or `allSalaries` would be a better choice.